library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java

```diff
@@ -236,6 +236,26 @@ public void setOnMarkerClickListener(OnMapMarkerClickListener listener) {
     webView.loadUrl(String.format(Locale.US, "javascript:removePolyline(%1$d);", polyline.getId()));
   }
 
+  @Override public <T> void addPolygon(AirMapPolygon<T> polygon) {
+    try {
+      JSONArray array = new JSONArray();
+      for (LatLng point : polygon.getPolygonOptions().getPoints()) {
+        JSONObject json = new JSONObject();
+        json.put("lat", point.latitude);
+        json.put("lng", point.longitude);
+        array.put(json);
+      }
+
+      webView.loadUrl(String.format("javascript:addPolygon(" + array.toString() + ", %1$d, %2$d, %3$d, %4$d);", polygon.getId(), (int) polygon.getPolygonOptions().getStrokeWidth(), polygon.getPolygonOptions().getStrokeColor(), polygon.getPolygonOptions().getFillColor()));
```

Comment:
can you wrap this line in multiple, please? It's too long. 
Also use `Locale.US` as the first argument to `String.format`